tests/resume.test.ts

```diff
@@ -0,0 +1,82 @@
+import { BadRequestError } from 'routing-controllers';
+import { ResumeModel } from '../models/ResumeModel';
+import { Config } from '../config';
+import { ControllerFactory } from './controllers';
+import { DatabaseConnection, PortalState, UserFactory } from './data';
+import { FileFactory } from './data/FileFactory';
+import Mocks from './mocks/MockFactory';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('upload resume', () => {
+  test('authenticated user can upload resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    await new PortalState()
+      .createUsers(member)
+      .write();
+
+    const resume = FileFactory.pdf(Config.file.MAX_RESUME_FILE_SIZE / 2);
+    const fileLocation = 'fake location';
+
+    const resumeController = ControllerFactory.resume(conn, Mocks.storage(fileLocation));
+    const response = await resumeController.updateResume(resume, member);
+    expect(response.error).toBe(null);
+    expect(response.resume.url).toBe(fileLocation);
+  });
+
+  test('updating resume deletes all previous resumes', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
```

Comment:
Can we verify that the storage mocks for deleting and uploading files properly get called here too?